spec/get_google_200.raw

```diff
@@ -0,0 +1,25 @@
+</script></div></body></html>
```

Comment:
Did we really need to use all of Google's content? Why not something like `about:blank's` content?